src/frontendFixes/Fixes/metaViewportScalableFix.js

```diff
@@ -10,8 +10,8 @@ const MetaViewportScalableFix = () => {
 	// Get the meta viewport tag.
 	const metaViewport = document.querySelector( 'meta[name="viewport"]' );
 	if ( metaViewport ) {
-		// check if it has scalable set to no.
-		if ( ! metaViewport.content.includes( 'user-scalable=no' ) ) {
+		// check if it has scalable set to no or 0.
+		if ( ! metaViewport.content.match( /user\-scalable\s*=\s*(no|0)/i ) ) {
```

Comment:
I branched had to rebase this branch when starting it so this change here is included unintentionally however it isn't in a new commit so it's an unrelated change but I can't avoid this here with a rebase.